### PR TITLE
zerocracy/farm#63 pm/staff/boosts.xsd

### PR DESCRIPTION
**Pull request by carlosmiranda**
zerocracy/farm#63: Added `pm/staff/boosts.xsd` and XML test files.

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this pull request

File: xsd/pm/staff/boosts.xsd [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ -->
+<xs:schema version="SNAPSHOT" xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    elementFormDefault="qualified" attributeFormDefault="unqualified">
+    <xs:include schemaLocation="http://datum.zerocracy.com/SNAPSHOT/xsd/basics.xsd"/>
+    <xs:complexType name="boost">
+        <xs:simpleContent>
+            <xs:extension base="xs:integer">
+                <xs:attribute name="id" type="xs:string" use="required" />
```
Comment (yegor256) [inline xsd/pm/staff/boosts.xsd]: @carlosmiranda this should be of type `jobID`, see `pm/scope/types.xsd`

File: xml/pm/staff/boosts/simple.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ -->
+<boosts updated="2017-07-27T04:06:33Z" version="0.1">
+  <boost id="2x">2</boost>
```
Comment (yegor256) [inline xml/pm/staff/boosts/simple.xml]: @carlosmiranda I would do it like this instead:

```xml
<boost job="gh:test/test#1">2</boost>
```

**Comment by yegor256** [comment]
@carlosmiranda see my comments above. Besides that, 1) you forgot to add `boosts.xsl`, and 2) I think it's better to put this into `pm/cost`, since this is related to the expenses project will have. Every payment will be boosted by the factor. Of course, awards will also be affected, but it's secondary. The primary purpose of the boost factor is to _pay_ more. My bad, I suggested `pm/staff` initially.

**Comment by carlosmiranda** [comment]
@yegor256 comments addressed. I did find the initial ticket confusing, should have asked. :)

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**